tasks/synopsys-detect-task/ts/runner/DetectScriptConfigurationRunner.ts

```diff
@@ -57,9 +58,25 @@ export class DetectScriptConfigurationRunner extends DetectRunner {
     async retrieveOrCreateArtifactFolder(fileName: string): Promise<string> {
         const workingDirectory = PathResolver.getWorkingDirectory() || ''
         const scriptFolder: string = PathResolver.combinePathSegments(workingDirectory, DetectADOConstants.SCRIPT_DETECT_FOLDER)
+        let isBlackDuckAccessible: boolean = true
+        const axios: AxiosInstance = Axios.create()
 
         try {
-            await DetectScriptDownloader.downloadScript(this.blackduckConfiguration.proxyInfo, fileName, scriptFolder)
+            const bdResponse = await axios.get(DetectScriptDownloader.DETECT_DOWNLOAD_URL)
+
+            if(bdResponse.status >= 200 && bdResponse.status <= 399) {
+                isBlackDuckAccessible = true;
+            } else {
+                logger.warn("https://detect.blackduck.com responded with an unexpected status code " + bdResponse.status + ". Please allow access through your firewall as https://detect.synopsys.com will be shutdown at the end of February 2025.")
```

Comment:
Can we remove the specific dates in this message and in the below one? I'd just remove the second sentence.